utils/fileSystem.py

```diff
@@ -194,5 +196,7 @@ def get_from_image_root(session_id):
                 images.append(os.path.join(image_path, f))
         else:
             pass
+    else:
+        raise Exception("Image directory does not exist")
```

Comment:
this function may throw an exception, which is not catched in the following files:
- action_scripts/changeFormat.py (line 14)
- action_scripts/convertToLowPoly.py (line 17)
- action_scripts/convertToPixelArt.py (line 16)
- action_scripts/igPanoSplit.py (line 14)
- action_scripts/resizeImage.py (line 14)
- dynamic_functions/dimensionFunctions.py (lines 9, 22, 35, 48)

Reply:
I will add the error handling for the dimension functions